packages/simulator/index.js

```diff
@@ -8,6 +8,11 @@ const kommuner = require('./streams/kommuner')
 const regions = require('./streams/regions')(kommuner)
 const { safeId } = require('./lib/id')
 const { readParameters } = require('./lib/fileUtils')
+const statistics = require('./lib/statistics')
+const elastic = require('./lib/elastic')
+elastic.createIndices()
```

Comment:
This is a promise that's not properly awaited. Should this maybe rather be something outside of this application that is run seperatly and before the application is started to prevent race conditions?